src/components/RecordingPlaybackOverlay/RecordingPlaybackOverlay.tsx

```diff
@@ -0,0 +1,37 @@
+import { LoadingOverlay, Stack, Text, ThemeIcon } from '@mantine/core';
+
+import { useSubscribeToEngineMode } from '@/hooks/topicSubscriptions';
+import { VideocamIcon } from '@/icons/icons';
+import styles from '@/theme/global.module.css';
+import { EngineMode, IconSize } from '@/types/enums';
+
+interface Props {
+  text?: string;
+}
+
+export function RecordingPlaybackOverlay({ text }: Props) {
```

Comment:
Is this component too sepcific?  I.e. do we want a more general "disable overlay" component rather than one specific to the session recording playback?

Reply:
If we need a more general component down the line we can always change it:)